src/modules/logger/logger.cpp

```diff
@@ -575,53 +575,60 @@ void Logger::add_common_topics()
 	// Note: try to avoid setting the interval where possible, as it increases RAM usage
 	add_topic("actuator_controls_0", 100);
 	add_topic("actuator_controls_1", 100);
-	add_topic("actuator_outputs", 100);
-	add_topic("airspeed");
+	add_topic("actuator_outputs", 200);
+	add_topic("airspeed", 200);
 	add_topic("att_pos_mocap", 50);
-	add_topic("battery_status", 300);
+	add_topic("battery_status", 500);
 	add_topic("camera_capture");
 	add_topic("camera_trigger");
-	add_topic("commander_state", 100);
+	add_topic("commander_state", 200);
 	add_topic("control_state", 100);
 	add_topic("cpuload");
-	add_topic("differential_pressure", 50);
-	add_topic("distance_sensor");
-	add_topic("ekf2_innovations", 50);
-	add_topic("esc_status", 250);
-	add_topic("estimator_status", 200); //this one is large
-	add_topic("gps_dump"); //this will only be published if gps_dump_comm is set
-	add_topic("input_rc", 100);
-	add_topic("optical_flow", 50);
+	add_topic("differential_pressure", 400);
+	add_topic("distance_sensor", 100);
+	add_topic("ekf2_innovations", 200);
+	add_topic("esc_status", 400);
+	add_topic("estimator_status", 200);
+	add_topic("input_rc", 200);
+	add_topic("manual_control_setpoint", 200);
+	add_topic("offboard_mission");
+	add_topic("onboard_mission");
+	add_topic("optical_flow", 100);
 	add_topic("position_setpoint_triplet", 200);
-	add_topic("rc_channels", 100);
```

Comment:
On second thought between rc_channels, input_rc, and manual_control_setpoint the most valuable for review is almost always manual_control_setpoint.
Would anyone be opposed to me changing it in flight review?